### ISSUE TITLE
Problem with missing overview of the day

### ISSUE THREAD

**Issue by xMordax** [projecthamster/hamster#457]
If I go on the overview of today's projects that I wrote down and click on the line at the bottom to see the summed summary of the hours spent I see for example example: 

![image](https://user-images.githubusercontent.com/27705384/65899585-55866400-e3b4-11e9-84f8-466f913686b6.png)

**Comment by ederag** [comment]
Thanks for the report.
Is the list going all the way down to the window border ? (not visible on the screenshot)
Are the missing Projects the ones with the smallest total duration ?

**Comment by xMordax** [comment]
Yes exactly, it is the end of the window and yes these are the one with the shortest duration
![image](https://user-images.githubusercontent.com/27705384/65977413-2a624a00-e472-11e9-8f54-21ef4f478b2a.png)

**Comment by ederag** [comment]
Thanks, confirmed.
We probably need a [`GtkPaned`](https://lazka.github.io/pgi-docs/Gtk-3.0/classes/Paned.html) to split the window between facts and stats.
And the `Totals` should be inserted in a separate `GtkScrolledWindow` as well.
In the meantime, you may change manually the height in this line:
https://github.com/projecthamster/hamster/blob/f5acb4de5f06e3f0334536c9f5f6f8f284aefdb3/src/hamster/overview.py#L353

**Comment by xMordax** [comment]
I tried installing the latest master branch but I am seeing the same problem...

**Comment by ederag** [comment]
Indeed. That's why the issue is still open.
We are working hard on other parts of the code,
and the dots feel like an unkind tentative to push.
As the milestone 3.1 implied, this won't be fixed in v3.0.
Have you even tried the workaround given above ?

**Comment by lalejand** [comment]
@xMordax did you manage to increase the height? I am having the same problem and I need to see all the projects. But I'm a noob about pulling and compiling my own hamster version. Did you manage to create a package?

**Comment by lalejand** [comment]
Help needed.
I need to see all projects in the stats area to be able to give feedback to my clients :/
Did someone compiled a version on Hamster with an increased height for the stats area?

**Comment by GeraldJansen** [comment]
AFAIK nobody has worked on this request. You could us Menu->Export to export to html, which opens a browser window on a report with the complete list.

**Comment by lalejand** [comment]
Thank you very much !!! I didn't know this functionality :)
